test/index.js

```diff
@@ -70,6 +70,7 @@ describe('markdown-it-link-attributes', function () {
 
     this.md.render('[link](https://google.com)')
 
+    // eslint-disable-next-line no-unused-expressions
```

Comment:
Can we just use `expect(spy).to.be.calledOnce // eslint-disable-line no-unused-expressions` instead of `next-line`?

Reply:
Sure :)

Reply:
Should I `--amend` the commit to #6fb8491?

Reply:
Doesn't matter, I'll probably just squash it anyway.